framework_tvb/tvb/adapters/simulator/simulator_fragments.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, pse_param1, pse_param2, prefix='', project_id=None):
 
         else:
             self.pse_param1_dt = DataTypeSelectField(h5.REGISTRY.get_index_for_datatype(pse_param1.type), self,
-                                                     name='pse_param1', required=True, label='pse_param1',
+                                                     name='pse_param1_guid', required=True, label='pse_param1_guid',
```

Comment:
Is this the label text as it appears in GUI?
Then it should be composed of separate words, not _